cerberus-web/src/main/resources/cerberus.yaml

```diff
@@ -80,60 +101,98 @@ cerberus:
 #        clientId: yourClientId
 #        clientSecret: yourClientSecret
 #        subdomain: yourSubDomain
+
+
+  auth.iam.kms:
+    # This should be the root user for the account that this service will be ran in, so that KMS keys can be managed by that user
+    rootUserArn: ~
+    # This should be some sort of roll that your team can assume, its ok to re-use the cms role arn here. This will be used when created the kms keys for auth to allow your team to manage them
```

Comment:
roll should be role

Reply:
"This will be used when created the kms keys for auth to allow your team to manage them" <- not sure what this sentence is saying